src/floki_selector_lexer.xrl

```diff
@@ -22,6 +22,7 @@ Rules.
 \({ODD}\)                            : {token, {pseudo_class_odd, TokenLine}}.
 \({EVEN}\)                           : {token, {pseudo_class_even, TokenLine}}.
 \({PSEUDO_PATT}\)                    : {token, {pseudo_class_pattern, TokenLine, remove_wrapper(TokenChars)}}.
+\({QUOTED}\)                         : {token, {pseudo_class_quoted, TokenLine, remove_wrapper(remove_wrapper(TokenChars))}}.
```

Comment:
I had to change the parsing of pseudo-class values because of this issue #100. I'm not sure but seems that this line does the same it was doing before I fix in #104 . Can you confirm this?